app/models/spotlight/resource.rb

```diff
@@ -30,10 +30,10 @@ def self.class_for_resource r
     end
 
     def to_solr
-      {
-        spotlight_resource_id_ssim: "#{(type.tableize if type) || self.class.to_s.tableize }:#{id}",
-        spotlight_resource_url_ssim: url
-      }
+      exhibit.solr_data.merge({
```

Comment:
I started out down that path but removed it because the `url` in the context of the uploaded image is a CarrierWave object or something like that.  It may be worth reevaluating that and just deleting out the fields that we don't want coming from the parent class.